tst/PortingAssistantExtensionUITests_FlaUI/TestBase.cs

```diff
@@ -0,0 +1,124 @@
+﻿using FlaUI.Core.AutomationElements;
+using FlaUI.Core.Tools;
+using FlaUI.UIA3;
+using IDE_UITest.UI;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace IDE_UITest
+{
+
+    public class TestBase: IDisposable
+    {
+        public const string VS2019ExePath = @"C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\Common7\IDE\devenv.exe";
```

Comment:
thinking about VS2022, is it possible to convert this to a generic VisualStudio path, and then pass into the constructor?

also maybe this visual studio path is something else we can add to the appsettings.json? this could give a bit of flexibility with things like VS community and such?

Reply:
Agree,  can move it to appsettings.